spec/spec_helper_acceptance.rb

```diff
@@ -0,0 +1,48 @@
+require 'beaker-rspec'
+
+# Various workarounds for Beaker's lack of Puppet 4 support.
+
```

Comment:
You can test against Puppet 4 with beaker, using https://github.com/puppetlabs/beaker-puppet_install_helper